docs/articles/licensing/index.rst

```diff
@@ -24,4 +22,10 @@ compliance tooling will be using in the future, and indeed already is
 used by tool makers such as WindRiver and Black Duck, they are included
 both for support for tooling and future proofing the format.
 
+.. toctree::
+    :caption: GPLv3 license in Automotive industry
```

Comment:
Why is Automotive in caps and other nouns not? It should be either all or none, please check with other chapters.

Reply:
It should also be "... in the Automotive ..."

Reply:
@SKumarDas please address this comment :)